### ISSUE TITLE
Sprint+ Crash on 1.19.4

### ISSUE THREAD

**Issue by sigmaclientwastaken** [KassuK1/BlackOut#14]
## Exception:
```
java.lang.NoSuchFieldError: field_3921
	at kassuk.addon.blackout.modules.SprintPlus.onTick(SprintPlus.java:45)
	at meteordevelopment.orbit.listeners.LambdaListener.call(LambdaListener.java:83)
	at meteordevelopment.orbit.EventBus.post(EventBus.java:47)
	at net.minecraft.class_310.handler$bgo000$meteor-client$onPreTick(class_310.java:9558)
	at net.minecraft.class_310.method_1574(class_310.java)
	at net.minecraft.class_310.method_1523(class_310.java:1180)
	at net.minecraft.class_310.method_1514(class_310.java:801)
	at net.minecraft.client.main.Main.main(Main.java:237)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:462)
```

## Sprint+ Settings:
SprintTime On
Mode Vanilla

## Possible Fix:
This is probably just a method having a different obfuscated name in 1.19.4 compared to the version the addon is compiled on, so most likely just updating to 1.19.4 will fix this.

### Edit
Just tested with SprintTime disabled, so that's the method with a different obfuscated name.

**Comment by CrytoPal** [comment]
Please send the entire log file!

**Comment by sigmaclientwastaken** [comment]
@CrytoPal the exception clearly shows what the error is.

**Comment by CrytoPal** [comment]
@sigmaclientwastaken The log would make it easier for us to understand because Blackout is currently 1.19.3 and it hasn't supported 1.19.4 yet.

**Comment by KassuK1** [comment]
This will be fixed when we release 1.0.0 (coming soon hopefully) also hello sigma
